Gemfile

```diff
@@ -80,6 +80,8 @@ gem 'image_processing', '>= 1.2'
 
 gem 'dotenv', groups: %i[development test]
 
+gem 'jquery-rails'
+gem 'jquery-ui-rails'
```

Comment:
Why are you using jQuery?

Reply:
Why not? 🤓

Nah, I forgot delete that gems. I was using different methods to send the emails before use SendGrid (like MailCatcher and others). Thx for the review.